### PR TITLE
Set up GitHub Actions for PR build

### PR DESCRIPTION
**Pull request by skylarnam**
- Add a yml file for a PR build step.
- This only includes the build and test step.

### CODE REVIEW

**Comment by tydunkel** [comment]
Will we be refactoring later to share yaml?

**Comment by tydunkel** [comment]
Is it possible to share any of the build steps, or will we need to maintain the builds separately?

**Comment by skylarnam** [comment]
> Will we be refactoring later to share yaml?

Do you think we could do that? Seems like the CI yaml we have and GitHub actions yaml have different formats.

File: .github/workflows/actions-pr.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    runs-on: windows-2022
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install .NET Core SDK
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 7.x
```
Comment (tydunkel) [inline .github/workflows/actions-pr.yml]: Is it possible to specify just to use the global.json version?

**Review by tydunkel — APPROVED** [review]
LGTM

File: .github/workflows/actions-pr.yml (hunk 1 above)
Comment (skylarnam) [reply .github/workflows/actions-pr.yml]: I'll look into it.

Comment (skylarnam) [reply .github/workflows/actions-pr.yml]: Added

**Review by tydunkel — APPROVED** [review]

**Pull request merged**